### PR TITLE
cordova-plugin-push-notifications --> v1.1

### PR DESCRIPTION
**Pull request by dannywillems**
- Some fixed has been done.
- Change the global structure for types (use a module with a type t instead of a type in the top-level).
- Add some functions.

And this version has been tested with Eliom.

### CODE REVIEW

File: packages/cordova-plugin-push-notifications/cordova-plugin-push-notifications.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,13 @@
+opam-version: "1.2"
+maintainer: "Danny Willems <contact@danny-willems.be>"
+authors: "Danny Willems <contact@danny-willems.be>"
+homepage: "https://github.com/dannywillems/ocaml-cordova-plugin-push"
+bug-reports:
+  "https://github.com/dannywillems/ocaml-cordova-plugin-push/issues"
+license: "LGPL-3.0 with OCaml linking exception"
+dev-repo: "https://github.com/dannywillems/ocaml-cordova-plugin-push"
```
Comment (dsheets) [inline packages/cordova-plugin-push-notifications/cordova-plugin-push-notifications.1.1/opam]: This needs `.git` appended to have the version control protocol recognized.

Comment (dannywillems) [inline packages/cordova-plugin-push-notifications/cordova-plugin-push-notifications.1.1/opam]: Oh yes I forgot!

**Comment by dannywillems** [comment]
I will continue to do some changes so I close this PR and I will reopen it later.

**Comment by avsm** [comment]
@dannywillems:

distfiles for a couple of these packages are now missing -- could they be restored?

```
#=== ERROR cordova-plugin-push-notifications.1.1 ==============================#
OpamStd.OpamSys.Exit(66)
#=== ERROR cordova-plugin-push-notifications.1.0 ==============================#
OpamStd.OpamSys.Exit(66)
```

**Comment by avsm** [comment]
@danmey the entry in the `url` file for these packages is a 404

**Comment by dannywillems** [comment]
@avsm It must not be the case. I remember I changed the repository name (ocaml-cordova-plugin-push --> ocaml-cordova-plugin-push-notifications) but GitHub does the redirection when I try to download the archive.

**Comment by avsm** [comment]
```
cordova-plugin-push-notifications.1.0 avsm$ cat  url 
http:
  "https://github.com/dannywillems/ocaml-cordova-plugin-push-notifications/archive/v1.0.tar.gz"
checksum: "52a5b3f2f88dde71d54f7af858864d25"
cordova-plugin-push-notifications.1.0 avsm$ curl -OL https://github.com/dannywillems/ocaml-cordova-plugin-push-notifications/archive/v1.0.tar.gz
cordova-plugin-push-notifications.1.0 avsm$ md5 v1.0.tar.gz 
MD5 (v1.0.tar.gz) = 32f788c22e3494f2da11ca7f5ff16e17
```

The checksums are wrong it seems?

**Comment by dannywillems** [comment]
Oh... It seems.

**Pull request closed without merging**